CedarJava/src/test/java/com/cedarpolicy/AuthTests.java

```diff
@@ -92,21 +92,21 @@ public void residual() {
         assumePartialEvaluation(() -> {
             try {
                 final PartialAuthorizationResponse response = auth.isAuthorizedPartial(q, policySet, new HashSet<>());
-                assertTrue(response.getDecision() == null);
-                assertEquals("p0", response.getResiduals().entrySet().iterator().next().getKey());
+                assertTrue(response.success.orElseThrow().getDecision() == null);
```

Comment:
How can the decision be `null` when the response is a success?

Reply:
This test already existed; I haven't changed its assertion, I only fixed the result.

As for the decision, in Cedar partial authorization, when there are unknowns (in this case, the request has the principal as unknown), a decision is not able to be made. Thus, the decision is null.